src/panels/config/devices/ha-config-devices-dashboard.ts

```diff
@@ -312,6 +338,19 @@ export class HaConfigDeviceDashboard extends LitElement {
         @row-click=${this._handleRowClicked}
         clickable
       >
+        ${this._activeFilterDomains(this.entries, this._searchParms)?.includes(
+          "zha"
+        )
+          ? html`<a href="/config/zha/add" slot="fab">
+              <mwc-fab
```

Comment:
Add `hasFab` to `hass-tabs-subpage-data-table` when you add a fab.

Also use `ha-fab` instead of `mwc-fab`.

Reply:
I think I am missing something.... I don't see ha-fab in the project

Reply:
needed a rebase